lib/src/base-styles/_nx-variables.scss

```diff
@@ -65,9 +65,14 @@
                             0 1px 2px 0 rgba(91, 95, 141, 0.2),
                             0 1px 1px 0 rgba(124, 128, 179, 0.2);
 
+  // NxDrawer
+  --nx-box-shadow-drawer: -34px 0 50px -32px hsla(0, 0%, 50%, 0.5);
+
   @include nx-dark-mode-helpers.dark-mode {
     --nx-border-grid: var(--nx-border-default);
     --nx-border-width: 2px;
+    // NxDrawer
+    --nx-box-shadow-drawer: -34px 0 50px -32px hsla(231, 41%, 10%, 0.75);
```

Comment:
I'm finding the mockups of the shadow a bit confusing but at any rate I'm not seeing indigo-10 involved anywhere.  Where did you get that from?

Reply:
> I'm finding the mockups of the shadow a bit confusing but at any rate I'm not seeing indigo-10 involved anywhere. Where did you get that from?

When I clicked onto the "Shadows" color 808080 it comes up with the HSLA value for `indigo-10`. I can confirm with Jason and ask them to adjust the box-shadow color to the HSLA value
![Screenshot 2023-05-01 at 9 43 27 AM](https://user-images.githubusercontent.com/89862727/235469370-8341abe0-d14e-48e2-a9fe-3f174563e4f3.png)

Reply:
> When I clicked onto the "Shadows" color 808080 it comes up with the HSLA value for indigo-10. I can confirm with Jason and ask them to adjust the box-shadow color to the HSLA value

They created a variable named "808080" which seems like a hex code which would be grey, and set its value to indigo? That definitely seems like a mistake one way or another.

Reply:
> They created a variable named "808080" which seems like a hex code which would be grey, and set its value to indigo? That definitely seems like a mistake one way or another.

Confirmed in slack that this should indeed by `indigo-10` : https://sonatype.slack.com/archives/CJUF9G7T2/p1683120481965319